pom.xml

```diff
@@ -73,8 +73,8 @@
     <version.io.github.openfeign.opentracing>0.1.0</version.io.github.openfeign.opentracing>
     <!-- spring-boot-starter-parent is a module of spring-boot-dependencies
         https://github.com/spring-projects/spring-boot/blob/master/spring-boot-starters/spring-boot-starter-parent/pom.xml -->
-    <version.org.springframework.boot>1.5.8.RELEASE</version.org.springframework.boot>
-    <version.org.springframework.cloud-spring-cloud-dependencies>Dalston.SR4</version.org.springframework.cloud-spring-cloud-dependencies>
+    <version.org.springframework.boot>1.5.12.RELEASE</version.org.springframework.boot>
+    <version.org.springframework.cloud-spring-cloud-dependencies>Edgware.SR3</version.org.springframework.cloud-spring-cloud-dependencies>
```

Comment:
It's probably fine, and I suppose we should expect another Edgware release, but Edgware.SR3 is using Spring Boot 1.5.10 (https://github.com/spring-cloud/spring-cloud-release/blob/vEdgware.SR3/spring-cloud-starter-parent/pom.xml#L8)

Reply:
True... It's probably best if we wait for a new Edgeware release. Depending on Spring Boot `1.5.12` is what we should be aiming for since it fixes multiple CVEs

Reply:
https://start.spring.io/ returns SB 1.5.12.RELEASE with Edgware.SR3

Reply:
Great to know!